SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -465,6 +465,45 @@ TArray<UClass*> GetAllSupportedClasses()
 	return Classes.Array();
 }
 
+void CopyWellKnownSchemaFiles()
+{
+	FString PluginDir = GetDefault<USpatialGDKEditorSettings>()->GetGDKPluginDirectory();
+
+	FString GDKSchemaDir = FPaths::Combine(PluginDir, TEXT("SpatialGDK/Extras/schema"));
```

Comment:
It looks like we're going to end up with file paths scattered all over our code base. We should probably move all the file paths to a single place at some point. This is definitely not blocking for this PR, just something I thought could be useful in the future.

Reply:
Yeah I think a good refactor is in order at some point.